### PR TITLE
Add more tests for --help function to bring up code coverage

### PR DESCRIPTION
**Pull request by campionfellin**
Tests `--help` for these functions:

```sh
run
logs
login
logout
create
clone
pull
push
status
open
deployments
undeploy
redeploy
versions
version
list
apis
help
```

Signed-off-by: campionfellin <campionfellin@gmail.com>

- [x] `npm run test` succeeds.
- [x] `npm run lint` succeeds.

### CODE REVIEW

File: tests/test.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import * as os from 'os';
+  import * as os from 'os';
```
Comment (grant) [inline tests/test.ts]: Not caught by linting?

**Comment by campionfellin** [comment]
Now we just need a way to figure out Coveralls, as it'll bounce around like this, decreasing on PRs and then going back up once merged into master and re-built.

File: tests/test.ts (hunk 1 above)
Comment (campionfellin) [reply tests/test.ts]: Nope, but nice catch

**Comment by campionfellin** [comment]
Good call, thanks for the idea. These tests aren't super helpful but raise coverage a bit. Kind of just "chore" work.

**Comment by grant** [comment]
You could even put `expectHelp` _inside_ the `describe` function since it's only used in that logical scope/closure.

**Comment by campionfellin** [comment]
Will do that later tonight and then merge.

**Comment by campionfellin** [comment]
Tests passed, merging now.

**Comment by grant** [comment]
Note: Please squash commits when merging. Each PR should ideally be 1 commit.

**Comment by campionfellin** [comment]
Noted, thanks.

**Pull request merged**